webpack.config.js

```diff
@@ -26,4 +25,18 @@ module.exports = {
     ],
   },
   devtool: 'source-map',
```

Comment:
We should not be using source-map for production mode.

Reply:
@nickpith @alex-bezek @mhemesath  I believe the source maps are still wanted for prod?

Reply:
yes? don't you always want source maps in production mode because thats when its minified and everything and we wantt to be able to debug things?
https://webpack.js.org/configuration/devtool/#devtool
![image](https://user-images.githubusercontent.com/8029578/58673607-bcfeab00-8311-11e9-98bd-0206dbb640ee.png)

![image](https://user-images.githubusercontent.com/8029578/58673610-c851d680-8311-11e9-901d-5ef9a41429e0.png)
It does have yes for production.
It does say this FYI
![image](https://user-images.githubusercontent.com/8029578/58673625-d7388900-8311-11e9-93ef-02b29d0c5557.png)